usr/share/rear/conf/default.conf

```diff
@@ -1526,22 +1526,24 @@ BACKUP_PROG_DECRYPT_OPTIONS="/usr/bin/openssl des3 -d -k "
 # (i.e. during "rear recover" a dynamic name does not make sense).
 BACKUP_PROG_ARCHIVE="${BACKUP_PROG_ARCHIVE:-backup}"
 # BACKUP_PROG_EXCLUDE is an array of strings that get written into a backup-exclude.txt file
-# that is used e.g. in 'tar -X backup-exclude.txt' to get things excluded from the backup.
+# which is used e.g. in 'tar -X backup-exclude.txt' to get things excluded from the backup.
```

Comment:
:+1: , it is a non-restrictive clause, therefore "which" is more correct. But there should be a comma before it: https://en.wiktionary.org/wiki/which#Usage_notes_2